### ISSUE TITLE
Error regarding sequence length

### ISSUE THREAD

**Issue by Pmishraa** [sortmerna/sortmerna#436]
Thank you for this great tool. I faced an error while running for one of my sequence., which is as  follows. Can you suggest what could be the possible ways to resolve this. 

[validate:291] [0;31mERROR[0m: Read ID: 24_1312876 Header: @VH00578:87:AAC2KM7HV:2:2407:54587:24324 1:N:0:ACGCTTCT Sequence length: 65727 > 30000 nt

**Comment by biocodz** [comment]
Yeah, that's an outdated relic limitation remaining since the first releases. We will modify this value and prepare the new release soon

**Comment by biocodz** [comment]
related to [issue 373](https://github.com/sortmerna/sortmerna/issues/373)

**Comment by JannesSP** [comment]
I encountered the same error when processing nanopore data.
Looking forward to the modification :D

**Comment by will-gebbie** [comment]
Also checking in as I encountered the same error on nanopore data.

**Comment by biocodz** [comment]
wonder if somebody could provide a small number of long reads (around a 100 would suffice) to test this case.

**Comment by biocodz** [comment]
Latest versions provide `--max_read_len` that allow changing the default of 30000nt
